dts/bindings/gpio/fcs,fxl6408.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2021 Abel Sensors
+# SPDX-License-Identifier: Apache-2.0
+
+description: FXL6408 I2C-based GPIO expander
+
+compatible: "fcs,fxl6408"
+
+include: [gpio-controller.yaml, i2c-device.yaml]
+
+properties:
+    label:
+      required: true
+
+    has-pud:
```

Comment:
Where is this property being used in the code?

Reply:
I've removed the has-pud part since it wasn't used in code.